.github/workflows/deployment-checks.yml

```diff
@@ -27,19 +27,118 @@ jobs:
         run: yarn ci:prepare-config
         env:
           MAINNET_RPC_ENDPOINT: ${{ secrets.MAINNET_RPC_ENDPOINT }}
+      - name: Check Mainnet Deployment Addresses
+        run: yarn check-network-deployments mainnet
+  check-polygon-deployments:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up environment
+        uses: ./.github/actions/setup
+      - name: Prepare Config
+        run: yarn ci:prepare-config
```

Comment:
This code is the same everywhere. Is there a way not to copy paste it?

Reply:
We can probably do something with composite actions. I'll take a look.